topology/probes/k8s/service.go

```diff
@@ -40,11 +40,18 @@ type serviceProbe struct {
 }
 
 func dumpService(srv *v1.Service) string {
-	return fmt.Sprintf("service{Name: %s}", srv.GetName())
+	return fmt.Sprintf("service{Namespace: %s, Name: %s}", srv.Namespace, srv.Name)
 }
 
 func (p *serviceProbe) newMetadata(srv *v1.Service) graph.Metadata {
-	return newMetadata("service", srv.Namespace, srv.GetName(), srv)
+	m := newMetadata("service", srv.Namespace, srv.Name, srv)
+	m.SetFieldAndNormalize("Ports", s.Spec.Ports)
```

Comment:
Did you mean `srv.Spec.Ports` instead of `s.Spec.Ports` (s is undefined)

Reply:
yes, fixed!